incgamma.go

```diff
@@ -0,0 +1,25 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import "github.com/gonum/mathext/internal/cephes"
+
+// IncGamma computes the incomplete Gamma integral
```

Comment:
add full-stop after "integral".

also add a brief definition.
scipy does it like so:
```
1 / gamma(a) * integral(exp(-t) * t**(a-1), t=0..x)
```

but we could reproduce the one coming from cephes (translating it into its latex-like form)